libs/langchain-openai/src/chat_models.ts

```diff
@@ -1150,30 +1148,20 @@ export class ChatOpenAI<
       );
     }
 
-    this.azureOpenAIApiInstanceName =
-      fields?.azureOpenAIApiInstanceName ??
-      getEnvironmentVariable("AZURE_OPENAI_API_INSTANCE_NAME");
+    this.azureOpenAIApiInstanceName = fields?.azureOpenAIApiInstanceName;
 
-    this.azureOpenAIApiDeploymentName =
-      fields?.azureOpenAIApiDeploymentName ??
-      getEnvironmentVariable("AZURE_OPENAI_API_DEPLOYMENT_NAME");
+    this.azureOpenAIApiDeploymentName = fields?.azureOpenAIApiDeploymentName;
 
-    this.azureOpenAIApiVersion =
-      fields?.azureOpenAIApiVersion ??
-      getEnvironmentVariable("AZURE_OPENAI_API_VERSION");
+    this.azureOpenAIApiVersion = fields?.azureOpenAIApiVersion;
 
-    this.azureOpenAIBasePath =
-      fields?.azureOpenAIBasePath ??
-      getEnvironmentVariable("AZURE_OPENAI_BASE_PATH");
+    this.azureOpenAIBasePath = fields?.azureOpenAIBasePath;
+
+    this.azureOpenAIEndpoint = fields?.azureOpenAIEndpoint;
 
     this.organization =
       fields?.configuration?.organization ??
       getEnvironmentVariable("OPENAI_ORGANIZATION");
 
```

Comment:
Isn'it it missing setting the endpoint property here, like the others?

Reply:
Will just remove entirely